### PR TITLE
test: [WIP]Report tests

### PR DESCRIPTION
**Pull request by Anurag810**
This PR contains the test cases for some reports to increase the over all test coverage.

### CODE REVIEW

File: erpnext/stock/report/stock_balance/test_stock_balance.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+def get_item_map(rows):
+
+	item_map = {}
+	for row in rows:
+		item_map[row[0]] = row
+
+	return item_map
+
+
+def get_item(item_code, qty):
```
Comment (rmehta) [inline erpnext/stock/report/stock_balance/test_stock_balance.py:48]: repetitive function (?)

**Review by rmehta — CHANGES_REQUESTED** [review]
DRY

File: erpnext/stock/report/stock_balance/test_stock_balance.py (hunk 1 above)
Comment (Anurag810) [reply erpnext/stock/report/stock_balance/test_stock_balance.py:48]: I just checked there is only one method for creating the item map.

**Comment by Anurag810** [comment]
will work on this later

**Pull request closed without merging**